genienlp/data_utils/example.py

```diff
@@ -45,20 +45,23 @@ class Example(NamedTuple):
     question_word_mask: List[bool]
     answer: List[str]
     answer_word_mask: List[bool]
+    context_plus_question: List[str]
+    context_plus_question_word_mask: List[bool]
 
     vocab_fields = ['context', 'question', 'answer']
 
     @staticmethod
     def from_raw(example_id: str, context: str, question: str, answer: str, tokenize, lower=False):
         args = [example_id]
-        for argname, arg in (('context', context), ('question', question), ('answer', answer)):
+        for argname, arg in (('context', context), ('question', question), ('answer', answer), ('question', context+' '+question)):
             words, mask = tokenize(arg.rstrip('\n'), field_name=argname)
```

Comment:
I see we pass 'question' here to the Task class, so for Almond tasks everything will be subword split (mask all True). Just confirming that's the expected behavior.

Reply:
Yes, masks will be all True except for words that are entities (start with capital letters).